### PR TITLE
music + other stuff

### PR DESCRIPTION
**Pull request by Pizzaandy**
#58 #53 #51

### CODE REVIEW

**Comment by Pizzaandy** [comment]
#58 #53 #51

**Comment by net8floz** [comment]
kinda sus

**Comment by Pizzaandy** [comment]
sussy b

File: Loungeware/Loungeware.yyp [hunk 1]
```diff
@@ -780,6 +780,7 @@
     {"CopyToMask":-1,"filePath":"datafiles/games/baku","resourceVersion":"1.0","name":"baku_mine.json","resourceType":"GMIncludedFile",},
     {"CopyToMask":-1,"filePath":"datafiles/games/jdllama","resourceVersion":"1.0","name":"jdllama_target.json","resourceType":"GMIncludedFile",},
     {"CopyToMask":-1,"filePath":"datafiles/games/katsaii","resourceVersion":"1.0","name":"katsaii_witchwanda.json","resourceType":"GMIncludedFile",},
+    {"CopyToMask":-1,"filePath":"datafiles","resourceVersion":"1.0","name":"config.dev.json","resourceType":"GMIncludedFile",},
```
Comment (net8floz) [inline Loungeware/Loungeware.yyp:783]: Unneeded entry

File: Loungeware/objects/NahooRunner_CTB/Create_0.gml [hunk 2]
```diff
@@ -166,7 +166,7 @@ nahoo_tracks =
 selected = choose(nahoo_tracks.track_base, nahoo_tracks.track_vw);
 nahoo_track = selected.trk;
 
-audio_play_sound(selected.song, 1, 0);
+sfx_play(selected.song, 1, false);
```
Comment (net8floz) [inline Loungeware/objects/NahooRunner_CTB/Create_0.gml:169]: is false the same as 0?

File: Loungeware/objects/___obj_menu_gallery/Draw_0.gml [hunk 3]
```diff
@@ -38,12 +38,12 @@ if (state == "normal"){
 		audio_sound_gain(sng_id, 0, 300);
 	}
 
-	// back button
-	if (KEY_SECONDARY_PRESSED){
+	// back button OR escape key
+	if (KEY_SECONDARY_PRESSED || keyboard_check_pressed(vk_escape)){
```
Comment (net8floz) [inline Loungeware/objects/___obj_menu_gallery/Draw_0.gml:42]: is it better to use "or" here so it matches the comment?

File: Loungeware/objects/___obj_menu_gallery/Draw_0.gml [hunk 4]
```diff
@@ -38,12 +38,12 @@ if (state == "normal"){
...
 		state = "fadeout_back";
 		fadeout_do =  back_to_main;
 		// stop sound
-			audio_sound_gain(sng_id, 0, 100);
+		audio_sound_gain(sng_id, 0, 100);
```
Comment (net8floz) [inline Loungeware/objects/___obj_menu_gallery/Draw_0.gml:46]: wow 100 is kind of big isn't it?

File: Loungeware/objects/NahooRunner_CTB/Create_0.gml (hunk 2 above)
Comment (Pizzaandy) [reply Loungeware/objects/NahooRunner_CTB/Create_0.gml:169]: yes that's the loops parameter

File: Loungeware/objects/___obj_menu_gallery/Draw_0.gml [hunk 5]
```diff
@@ -79,7 +79,7 @@ var _list_margin_x = 8;
 var _list_margin_y = _div_y1;
 var _list_x = _list_margin_x;
 var _list_line_sep = 15;
-var _list_y = _list_margin_y + scroll_y + _list_line_sep*0.5;
+var _list_y = floor(_list_margin_y + scroll_y + _list_line_sep*0.5);
```
Comment (net8floz) [inline Loungeware/objects/___obj_menu_gallery/Draw_0.gml:82]: don't you mean ceil()

Comment (Pizzaandy) [reply Loungeware/objects/___obj_menu_gallery/Draw_0.gml:82]: no 😎

File: Loungeware/objects/___obj_pause/Create_0.gml [hunk 6]
```diff
@@ -5,6 +5,12 @@ microgame_current_metadata = ___MG_MNGR.microgame_current_metadata;
 instances_deactivated = false;
 jam_id = noone;
 
+input_cooldown = 0;
```
Comment (net8floz) [inline Loungeware/objects/___obj_pause/Create_0.gml:8]: this is a lot of newlines you can actually just write all vars on one line to save costs on whitespace

File: Loungeware/objects/___obj_pause/Step_0.gml [hunk 7]
```diff
@@ -41,15 +41,29 @@ if (state == "wait"){
 
 if (state == "paused"){
 	
-	var _vmove = -KEY_UP_PRESSED + KEY_DOWN_PRESSED;
+	var _vmove = -KEY_UP + KEY_DOWN;
```
Comment (net8floz) [inline Loungeware/objects/___obj_pause/Step_0.gml:44]: Is it needed to yell at the code?

File: Loungeware/sounds/___sng_zandintro/___sng_zandintro.yy [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {
-  "compression": 0,
+  "compression": 2,
```
Comment (net8floz) [inline Loungeware/sounds/___sng_zandintro/___sng_zandintro.yy:2]: wow

**Comment by torontofangirl** [comment]
as you can see Loungeware is very professional

**Pull request merged**